libs/web3/src/lib/use-vega-transaction-toasts.spec.tsx

```diff
@@ -59,20 +60,64 @@ jest.mock('@vegaprotocol/orders', () => {
     ...jest.requireActual('@vegaprotocol/orders'),
     useOrderByIdQuery: jest.fn(({ variables: { orderId } }) => {
       if (orderId === '0') {
+        const data: OrderByIdQuery = {
+          orderByID: {
+            id: '0',
+            side: 'SIDE_BUY',
+            size: '10',
+            remaining: '10',
+            timeInForce: 'TIME_IN_FORCE_FOK',
+            type: 'TYPE_MARKET',
+            price: '1234',
+            createdAt: new Date(),
+            status: 'STATUS_ACTIVE',
+            market: { id: 'market-1' },
+          },
+        } as OrderByIdQuery;
         return {
-          data: {
-            orderByID: {
-              id: '0',
-              side: 'SIDE_BUY',
+          data,
+        };
+      } else {
+        return { data: undefined };
+      }
+    }),
+    useStopOrderByIdQuery: jest.fn(({ variables: { stopOrderId } }) => {
+      if (stopOrderId === '0') {
+        const data: StopOrderByIdQuery = {
+          stopOrder: {
+            id: '0',
+            ocoLinkId: null,
+            expiresAt: null,
+            expiryStrategy: null,
+            triggerDirection: 'TRIGGER_DIRECTION_RISES_ABOVE',
+            status: 'STATUS_CANCELLED',
+            createdAt: '2023-08-03T07:12:36.325927Z',
+            updatedAt: null,
+            partyId: 'party-id',
+            marketId: 'market-1',
+            trigger: {
+              price: '1234',
+              __typename: 'StopOrderPrice',
+            },
+            submission: {
+              marketId: 'market-1',
+              price: '1234',
               size: '10',
+              side: 'SIDE_SELL',
               timeInForce: 'TIME_IN_FORCE_FOK',
+              expiresAt: null,
               type: 'TYPE_MARKET',
-              price: '1234',
-              createdAt: new Date(),
-              status: 'STATUS_ACTIVE',
-              market: { id: 'market-1' },
+              reference: '',
+              peggedOrder: null,
+              postOnly: false,
+              reduceOnly: true,
+              __typename: 'OrderSubmission',
             },
+            __typename: 'StopOrder',
           },
+        } as StopOrderByIdQuery;
```

Comment:
Same as above. Using the correct enums should let you not need the type cast

Reply:
The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables